sky/utils/kubernetes/nginx-deploy.yaml

```diff
@@ -0,0 +1,671 @@
+apiVersion: v1
```

Comment:
Can we use the installation command from the [guide](https://kubernetes.github.io/ingress-nginx/deploy/#bare-metal-clusters)?

```
kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.8.2/deploy/static/provider/baremetal/deploy.yaml
```

I noticed some fields are changed - we can [patch](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/)/update them after creating the official deployment?  That way we don't have to maintain this + changes required for SkyPilot are clear.

Reply:
done